content/_data/articles.js

```diff
@@ -0,0 +1,31 @@
+const groq = require('groq');
+const client = require('../../utils/sanityClient');
+const { hero, heroAlt } = require('../../utils/imageGroq');
+const toMarkdown = require('@sanity/block-content-to-markdown');
+const _ = require('lodash');
+
+module.exports = async function() {
+  return await client.fetch(groq`
+    *[_type == "article"
+      && !(_id in path("drafts.**"))
+    ]{
+      title,
+      subtitle,
+      summary,
+      body,
+      "slug": slug.current,
+      "tags": tags[]->title,
+      "date": _createdAt,
+      "category": category->title,
+      ${hero},
+    }
+  `).then(data => {
+    data.map(page => {
+      page.body = page.body ? toMarkdown(page.body) : '';
+      page.hero = heroAlt(page.hero);
+      return page;
+    });
+
+    return _.groupBy(data, item => item.category);
+  });
+};
```

Comment:
This should work fine, but usually I would expect to see either `async/await` _or_ `.then()` syntax -- in this case it's redundant. Anything after `await` is essentially running within a `.then(...)` context. For example, I might expect e.g.:

```suggestion
module.exports = async function() {
  const data = await client.fetch(groq`
    *[_type == "article"
      && !(_id in path("drafts.**"))
    ]{
      title,
      subtitle,
      summary,
      body,
      "slug": slug.current,
      "tags": tags[]->title,
      "date": _createdAt,
      "category": category->title,
      ${hero},
    }
  `);

  const pages = data.map(page => {
    page.body = page.body ? toMarkdown(page.body) : '';
    page.hero = heroAlt(page.hero);
    return page;
  });

  return _.groupBy(pages, page => page.category);
};
```

Reply:
Same goes for most other uses of `await` throughout this PR.

Reply:
Thank you! I still haven't totally got my mind around async/await - this is real helpful.